golang/trace/zipin_tracereporter_test.go

```diff
@@ -267,3 +269,63 @@ func BenchmarkBuildThrift(b *testing.B) {
 		buildZipkinSpan(span, annotations, nil, endpoint)
 	}
 }
+
```

Comment:
add tests for `TestBuildBinaryAnnotations` (you can use `buildBinaryAnnotation` in that test)